### PR TITLE
Use apache2 cookbook v7.1.0

### PR DESCRIPTION
**Pull request by madwort**
### Description

Implements support for apache2 cookbook v7

### Issues Resolved

https://github.com/sous-chefs/passenger_apache2/issues/65

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -19,4 +19,4 @@
 source_url 'https://github.com/chef-cookbooks/passenger_apache2'
 issues_url 'https://github.com/chef-cookbooks/passenger_apache2/issues'
 
-chef_version '>= 12.1'
+chef_version '>= 13.9'
```
Comment (madwort) [inline metadata.rb]: apache2 7.1 requires chef 13.9, so may as well make that explicit here too.

File: metadata.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 recipe 'passenger_apache2::default', 'Installs Passenger as an Apache module'
 recipe 'passenger_apache2::mod_rails', 'Enables Apache module configuration for passenger module'
 
-depends 'apache2', '>= 2.0'
+depends 'apache2', '~> 7.1'
```
Comment (tas50) [inline metadata.rb]: You probably want to just make this >= 7.1. Aggressive version constraints have been super problematic historically in community cookbooks.

**Comment by tas50** [comment]
Paging @damacus for Apache2 guidance

File: metadata.rb (hunk 1 above)
Comment (tas50) [reply metadata.rb]: Yep this is a totally safe version of Chef to assume at this point

File: metadata.rb (hunk 2 above)
Comment (madwort) [reply metadata.rb]: Ok. Yeah, I don't think you can currently win with this -> in our team I think we've had issues both with cookbook version constraints being too lax and too aggressive. I can see that it's more flexible to make them less aggressive here & pass on the responsibility to downstream users to pin more aggressively as/when required, though, so I've changed it as you suggest.

File: metadata.rb (hunk 1 above)
Comment (madwort) [reply metadata.rb]: 👍

**Comment by Xorima** [comment]
Hey @madwort 

What do we need to do to get this out of a draft and into our codebase from your perspective? Happy to help if you need assistance

Feel free also to drop by the #sous-chefs channel in slack, https://community-slack.chef.io/

**Comment by madwort** [comment]
Hey @Xorima, so, I think it basically works & we were unblocked for our internal project, although that project has now been deferred, so I don't have much time to do further work on this at the moment. 

It needs all the PR boilerplate doing - running tests, deciding what (if any) further tests are needed, getting a second opinion on whether it's the right way to fix it.

**Comment by Xorima** [comment]
hey @madwort 

That sounds good, if you could rebase against master we could at least move this on to ready for review, if you have time and are ok with that?

**Comment by madwort** [comment]
alrighty, did a quick rebase against master

**Comment by Xorima** [comment]
merged, it looks all good to me so why not 👍

**Pull request merged**